site/docs/relations/logical_relations.md

```diff
@@ -474,6 +474,32 @@ Write definition types are built by the community and added to the specification
 | Format   | Enumeration of available formats. Only current option is PARQUET. | Required |
 
 
+## Update Operator
+
+The update operator is an operator that consumes one input and writes the column updates to a storage. 
+
+| Signature            | Value                                 |
+| -------------------- |---------------------------------------|
+| Inputs               | 1                                     |
```

Comment:
Technically this new operator doesn't have an input (besides the table name).

Reply:
We can consider the table name on which the operator applied as input?
0 inputs for a update operator sounds strange.

Reply:
Inputs are traditionally other relations.  You could say zero and add a comment explaining that it does get data from somewhere.  The closest analog we have for existing relations is ReadRel.

Reply:
ok fixed it. Please take a look.